src/pages/search/Search.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useRef } from 'react'
+import styled from 'styled-components'
+import { Box } from 'atomic-layout'
+import debounce from 'lodash.debounce'
+import { Link } from 'react-router-dom'
+import Grid from '../../atoms/Grid'
+
+const StyledInput = styled.input`
+  background-color: #282828;
+  color: #fff;
+  width: 100%;
+  padding: 16px;
+  border: none;
+  font-size: 36px;
+  line-height: 44px;
+  letter-spacing: 0.02em;
+  font-weight: bold;
+
+  @media (max-width: 768px) {
```

Comment:
I suggest you use `Layourt.breakpoints` instead, so your media queries are consistent.

```js
import Layout from 'atomic-layout'

// ...

@media (max-width: ${Layout.breakpoints.md.minWidth}) {
  // ...
}
```